test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetTrustTests.cs

```diff
@@ -0,0 +1,564 @@
+            }
+        }
+
+        [CIOnlyFact]
+        public async Task DotnetTrust_AuthorAction_RelativePathConfileFile_WithoutExistingTrustedSignersSection_Succeeds()
+        {
+            // Arrange
+            var nugetConfigFileName = "NuGet.Config";
+            var package = new SimpleTestPackageContext();
+
+            using (SimpleTestPathContext pathContext = _msbuildFixture.CreateSimpleTestPathContext())
+            using (MemoryStream zipStream = await package.CreateAsStreamAsync())
+            using (TrustedTestCert<TestCertificate> trustedTestCert = SigningTestUtility.GenerateTrustedTestCertificate())
+            {
+                string certFingerprint = SignatureTestUtility.GetFingerprint(trustedTestCert.Source.Cert, HashAlgorithmName.SHA256);
+                string signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(trustedTestCert.Source.Cert, package, pathContext.PackageSource);
+                var config = $@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                      <packageSources>
+                        <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+                        <clear />
+                        <add key=""NuGetSource"" value=""{pathContext.PackageSource}"" />
+                       </packageSources>
+                      <config>
+                        <add key=""signaturevalidationmode"" value=""accept"" />
+                      </config>
+                    </configuration>";
+                SettingsTestUtils.CreateConfigurationFile(nugetConfigFileName, pathContext.WorkingDirectory, config);
+                var nugetConfigPath = Path.Combine(pathContext.WorkingDirectory, nugetConfigFileName);
+
+                // Act
+                CommandRunnerResult resultAdd = _msbuildFixture.RunDotnet(
+                    pathContext.SolutionRoot,
+                    $"nuget trust author nuget {signedPackagePath} --configfile ..{Path.DirectorySeparatorChar}{nugetConfigFileName}");
+
+                // Assert
+                resultAdd.Success.Should().BeTrue();
+                resultAdd.AllOutput.Should().Contain(string.Format(CultureInfo.CurrentCulture, _successfulAddTrustedSigner, "author", "nuget"));
+
+                string expectedResult = SettingsTestUtils.RemoveWhitespace($@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                      <packageSources>
+                        <!--To inherit the global NuGet package sources remove the < clear /> line below-->
+                        <clear/>
+                        <add key = ""NuGetSource"" value = ""{pathContext.PackageSource}""/>
+                       </packageSources >
+                      <config>
+                        <add key = ""signaturevalidationmode"" value = ""accept""/>
+                      </config>
+                      < trustedSigners>
```

Comment:
typo: extra space in the `<trustedsigner>` element.

Reply:
See other [comment](https://github.com/NuGet/NuGet.Client/pull/3989#discussion_r619328162).